code-challenges.test.js

```diff
@@ -24,39 +24,115 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
+/* Psuedo Code:
+- I want to declare a function called sentenceCased
+- sentenceCased will take an array of words
+- using (.map) to iterate over each string inside the array
+- call on the first index [0] give it the (.toUpperCase) then I want to add my substring(1) to be not capitalized after
+- the return will be joined and then split to turn it back to an array separated by commas
+*/
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("sentenceCased", () => {
+  it("takes in an array of words and returns an array with all the words capitalized", () => {
+    expect(sentenceCased(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(sentenceCased(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
+// ReferenceError: sentenceCased is not defined
+
+// // b) Create the function that makes the test pass.
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+const sentenceCased = (array) => {
+  let eachItem = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1) //substring returns part of the string between given start and end index; only given a start will end at last index
+  })
+  return eachItem.join(" ").split(" ")
+}
+// console.log(sentenceCased(randomNouns1)) //["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// console.log(sentenceCased(randomNouns2)) //["temperature", "database", "chopsticks", "mango", "deduction"]
 
-// b) Create the function that makes the test pass.
 
 
+//QUESTION #3
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+/* Psuedo Code:
+- First I want to create a function that will find the first index of the first vowel in a string
+- Name function firstVowel
+- Declare what vowels are
+- A for loop will return the index in which the first vowel is in
+- I want the loop to go through the length of the string and see which inncludes the given vowels
+- Then return the first index that our given strings have a vowel at
+*/
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("firstVowel", () => {
+  it("takes in a string and returns the first index of a vowel", () => {
+    expect(firstVowel(vowelTester1)).toEqual(1)
+    expect(firstVowel(vowelTester2)).toEqual(0)
+    expect(firstVowel(vowelTester3)).toEqual(2)
+  })
+})
```

Comment:
Tests are looking great!  In the future, I would recommend moving your test variables into your test to confine them to the scope in which they are be used.